src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -201,12 +201,30 @@ export default defineConfig({
 
 ### `imageService`
 
-`imageService: "passthrough" | "cloudflare"`
+`imageService: "passthrough" | "cloudflare" | "compile"`
 
 Determines which image service is used by the adapter. The adapter will default to `passthrough` mode when an incompatible image service is configured. Otherwise, it will use the globally configured image service:
 
 * **`cloudflare`:** Uses the [Cloudflare Image Resizing](https://developers.cloudflare.com/images/image-resizing/) service.
 * **`passthrough`:** Uses the existing [`noop`](/en/guides/images/#configure-no-op-passthrough-service) service.
+* **`compile`:** Uses Astro's default service (sharp) to optimize images during build.
```

Comment:
```suggestion
* **`compile`:** Uses Astro's default service (sharp) to optimize images on pre-rendered routes at build time. This disables image optimization during SSR for pages rendered on-demand.
```
I'm wondering whether we can just say something like this instead of the caution. Would something of this style this cover it?

Reply:
@alexanderniebuhr Changed my suggestion above. What do you think? This also matches the others that simply say "Uses service X" (and don't say "to optimize images") so that the focus can be on when it does/doesn't work.